src/Compilers/CSharp/Portable/Binder/Binder_Constraints.cs

```diff
@@ -137,14 +138,18 @@ internal partial class Binder
 
                         if (i != 0)
                         {
-                            diagnostics.Add(ErrorCode.ERR_RefValBoundMustBeFirst, syntax.GetFirstToken().GetLocation());
+                            if (!reportedOverrideWithConstraints)
+                            {
+                                reportNotFirstOrCannotCombine(diagnostics, syntax.GetFirstToken().GetLocation(), ErrorCode.ERR_RefValBoundMustBeFirst, MessageID.IDS_Class, firstTypeConstraintString);
```

Comment:
>reportNotFirstOrCannotCombine(diagnostics, syntax.GetFirstToken().GetLocation(), ErrorCode.ERR_RefValBoundMustBeFirst, MessageID.IDS_Class, firstTypeConstraintString); [](start = 32, length = 167)

What is going to be reported for a scenario when a ```class``` constraint is specified twice?

Reply:
Simplified and added a test case.

---
In reply to: [441187653](https://github.com/dotnet/roslyn/pull/45147#discussion_r441187653) [](ancestors = 441187653)